### ISSUE TITLE
0.7.5 fails to build with "error: can't copy 'icons/add_node.png': doesn't exist or not a regular file"

### ISSUE THREAD

**Issue by hotice** [syncthing/syncthing-gtk#195]
The latest syncthing-gtk 0.7.5 fails to build, with the following error:
```
error: can't copy 'icons/add_node.png': doesn't exist or not a regular file
```
And indeed, the icons/add_node.png file doesn't exist, there's just a icons/add_node.svg icon. This may also be the case with other icons.

You can find the complete build log here: https://launchpadlibrarian.net/216142298/buildlog_ubuntu-wily-amd64.syncthing-gtk_0.7.5-1~webupd8~wily0_BUILDING.txt.gz

**Comment by XRevan86** [comment]
```diff
--- a/setup.py
+++ b/setup.py
@@ -62,13 +62,13 @@ if __name__ == "__main__" :
 		('share/syncthing-gtk', glob.glob("*.glade") ),
 		('share/syncthing-gtk', glob.glob("scripts/syncthing-plugin-*.py") ),
 		('share/syncthing-gtk/icons', [
-				"icons/%s.png" % x for x in (
-					'add_node', 'add_repo', 'address',
-					'announce', 'clock', 'compress', 'cpu', 'dl_rate',
-					'eye', 'folder', 'global', 'home', 'ignore', 'lock',
-					'ram', 'restart', 'settings', 'shared', 'show_id',
-					'shutdown', 'show_id', 'sync', 'thumb_up',
-					'up_rate', 'version'
+				"icons/%s" % x for x in (
+					'add_node.svg', 'add_repo.svg', 'address.svg', 'announce.svg',
+					'clock.svg', 'compress.svg', 'cpu.svg', 'dl_rate.svg',
+					'eye.svg', 'folder.svg', 'global.svg', 'home.svg', 'ignore.svg',
+					'lock.svg', 'ram.svg', 'shared.svg', 'show_id.svg', 'show_id.svg',
+					'sync.svg', 'thumb_up.svg', 'up_rate.svg', 'version.svg',
+					'restart.png', 'settings.png', 'shutdown.png'
 			)]),
 		('share/icons/hicolor/64x64/emblems', glob.glob("icons/emblem-*.png") ),
 		('share/pixmaps', ["icons/syncthing-gtk.png"]),
```

**Comment by kozec** [comment]
Aww, snap, I knew I'm forgetting something :(

XRevan's patch looks good but I'll make buildable release right after I get home.

**Comment by kozec** [comment]
Probem should be fixed with [v0.7.5.1](https://github.com/syncthing/syncthing-gtk/releases/tag/v0.7.5.1)
